packages/flutter/lib/src/material/text_field.dart

```diff
@@ -721,8 +721,9 @@ class _TextFieldState extends State<TextField> with AutomaticKeepAliveClientMixi
         from: details.globalPosition,
         cause: SelectionChangedCause.forcePress,
       );
-      if (_shouldShowSelectionToolbar)
+      if (_shouldShowSelectionToolbar) {
         _editableTextKey.currentState.showToolbar();
```

Comment:
Would 2 long presses trigger the bug, i.e. should this be toggleToolbar?

Reply:
Yes, good catch!